Mastodon/Scene/Onboarding/PickServer/MastodonPickServerViewController.swift

```diff
@@ -141,6 +141,13 @@ extension MastodonPickServerViewController {
             )
         }
         .store(in: &disposeBag)
+        
+        viewModel.scrollToTop
```

Comment:
This kind of feels ... weird for something as simple as scrolling to the top. Usually I'd have used a method for this, but I still live in the old school object oriented world :D